src/imageio.cs

```diff
@@ -2356,4 +2368,17 @@ interface CGImageAuxiliaryDataInfo {
 		NSData Data { get; set; }
 		NSDictionary DataDescription { get; set; }
 	}
+
+	[Partial]
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	interface CGImageAnimationOptions {
+		[Field ("kCGImageAnimationDelayTime")]
+		NSString DelayTime { get; }
+
+		[Field ("kCGImageAnimationLoopCount")]
+		NSString LoopCount { get; }
```

Comment:
Is it an NSNumber of some kind? Can we use BindAs?

Reply:
Likewise, from the docs: `const CFStringRef kCGImageAnimationLoopCount;`

Reply:
Yet we have in the header:
```
/* The value of this key overrides the `loop count' specified by the image
+ * Value is a CFNumber; kCFNumberPositiveInfinity may be used.
+ */
+IMAGEIO_EXTERN const CFStringRef kCGImageAnimationLoopCount  IMAGEIO_AVAILABLE_STARTING(10.15, 13.0);
```
So it is a CFNumber.

Reply:
same, I'm an idiot.